README.md

```diff
@@ -11,6 +12,88 @@ This document contains guidelines for setup and running of RISC-V Architecture F
+
+| Extension | RV32 | RV64 | Notes |
+| --- | --- | --- | --- |
+| **Unprivileged** |
+| I | | | Integer base | 
+| M | | | Mul/div |
+| F | | | Floating-point |
+| D | | | Double-precision floating-point |
+| Zfh |  | | Half-precision floating-point |
+| Zfhmin |  | | Half-precision transfers/converts |
+| Zfa | | | Additonal floating-point: F, D, Zfh |
+| Zaamo | | | A extension: atomic memory operations |
+| Zalrsc | | | A extension: load reserved/store conditional |
+| Zba | | | B extension: address generation |
+| Zbb | | | B extension: basic bit manipulation |
+| Zbc | | | B extension: carry-free multiplication |
+| Zbs | | | B extension: single-bit operations |
+| Zca | | | Compressed instructions |
+| Zcb | | | Additional compressed instructions |
+| Zcf |x | | RV32 compressed single-precision fp |
+| Zcd | | | Compressed double-precision fp |
+| Zbkb | | | Basic bit manipulation for crypto |
+| Zbkc | | | Carry free multiplication for crypto | 
+| Zbkx |  | | Crossbar permutations for crypto |
+| Zknd | | | AES decryption |
+| Zkne | | | AES encryption |
+| Zknh | | | SHA2 hash |
+| Zicond | | | Conditional zero |
+| Zicbom | | | Cache block management, architecturally invisible |
+| Zicboz | | | Cach block zero |
+| Zicsr | | | CSR read/write/set/clear |
+| Zifencei | | | Instruction/data synchronization |
+| **Privileged** |
+| Zicsr | | | Excercise CSRs in each mode: M, S, U, F |
+| Exceptions | | | Exceptions: M, S, U, F, Zc, ZicboU, ZicboS, Zalrsc, Zaamo |
+| Interrupts | | | Interrupts: M, S, U, Sstc |
+| VM | SV32 | SV39/48 | Virtual Memory: Svbare, Sv32/39/48, Svade, Svadu, Sstvecd, Svinval, Svnapot |
+| PMP | | | Physical memory protection
+| Endian | | | Big vs. little endian: M, S, U |
+| Zicntr| | | Counters and performance monitors |
+| **Miscellaneous** |
+| Fences | | | Tested in Zicsr |
+| Zihintpause | | | Tested in Zicsr |
+| Zicclsm | | | Misaligned access support is implicitly tested through accesses |
+| Ss1p12 | | | Implicit in Zicsr |
+| Sstvala | | | stval implicity tested through exceptions |
+| Sscounterenw | | | Writable scounteren tested through Zicntr |
+| Ssu64xl | | | sstatus.UXL tested through Zicsr |
+| **Untested** |
+| PMA | | | Implementation dependent, not architectural |
+| Ziccif | | | Main memory cachability and coherence part of PMA |
+| Ziccrse | | | RsrvEventual part of PMA |
+| Ziccamoa | | | AMOArithmetic part of PMA |
+| Svbpmt | | x | Uncachable regions not testable architecturally |
+| Za64rs | | | Reservation set size not tested |
+| Zi64b | | | Cache block size not tested architecturally |
+| Zicbop | | | Cache block prefetch architecturally invisible |
+| Zkt | | | Data-independent execution time architecturally invisible |
+| Zkr | | | Entropy source hard to test architecturally |
+| Ssccptr | | | Hardware page table reads part of PMA |
+
+Notes:
+As of 12/7/24, atomic, CBO, Zifencei, crypto, Sscofpmf not implemented and privileged tests are in progress
```

Comment:
As currently formatted this entire section appears as one line with no punctuation. It should probably use bullet points instead.